plugins/destination/duckdb/client/migrate.go

```diff
@@ -215,7 +214,16 @@ func (c *Client) createTableIfNotExist(table *schema.Table) error {
 			sb.WriteString(",")
 		}
 	}
-
+	if len(pks) > 0 && c.enabledPks() {
```

Comment:
```suggestion
	if pks := table.PrimaryKeys(); len(pks) > 0 && c.enabledPks() {
```

No blocking but we could do something like this